src/calculateRentalCost.js

```diff
@@ -5,6 +5,13 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
```

Comment:
Replace the hard-coded value `40` with a constant to represent the daily rental cost and `20` with a constant to represent the discount for renting 3 or more days.